File: BOJ/#4344 평균은 넘겠지/평균은 넘겠지.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int C; cin>>C;
    for(int tc=0; tc<C; tc++) {
        int N; cin>>N;

        vector<int> scores;
        int sum = 0;
        for(int i=0; i<N; i++) {
            int n; cin>>n;
            scores.push_back(n);
            sum += n;
        }

        double avg = (double)sum/(double)N;
        int cnt = 0;
        for(int score : scores)
            if(score > avg) cnt++;

        printf("%.3f%\n", (double)cnt/(double)N*100.);
    }
}

File: BOJ/#4344 평균은 넘겠지/평균은 넘겠지.py
for tc in range(int(input())):
    scores = list(map(int, input().split()))
    N = scores[0]
    scores = scores[1:]

    avg = sum(scores)/N
    cnt = 0
    for score in scores:
        if score > avg: cnt += 1

    print("%.3f"%(cnt/N*100), end='')
    print('%')
